Release Notes CD - March, 30, version 789xx (#1156)
harness/developer-hub: release-notes/continuous-delivery.md

- The Google Artifact Image **Version** drop-down options were not visible in the **Google Artifact Registry Repository** template dialog. (CDS-55094)

  This issue is fixed. Google Artifact Image version options are now visible for Google Artifact Registry (GAR) artifact source template.
- The OCI Helm connector connection test failed for the Helm repository URL with port number: `public.ecr.aws:443` with anonymous credentials. (CDS-54066)

  This issue is fixed. We now support the following URL types for the OCI Helm connector.

  * URL without the `oci://` prefix. For example, `public.ecr.aws`.
  * URL with the `oci://` prefix. For example, `oci://public.ecr.aws`.
  * URL with port number. For example, `public.ecr.aws:443`.
  * URL with the `oci://` prefix and port number. For example, `oci://public.ecr.aws:443`. 
- Users were able to save the [Kubernetes apply step](https://developer.harness.io/docs/continuous-delivery/cd-technical-reference/cd-k8s-ref/kubernetes-apply-step/) template without file paths in the template library using YAML. (CDS-53961)

  This issue is fixed. You can no longer configure empty file paths in the apply step template.

> - Unable to create and view default values when configuring services, environments, templates, and pipelines. (CDS-53919, ZD-39998, ZD-40031, ZD-41197, ZD-41889)

Reviewer: ```suggestion
- Users were unable to create or edit the runtime input default values when configuring services, environments, templates, and pipelines. (CDS-53919, ZD-39998, ZD-40031, ZD-41197, ZD-41889)
```

>   This issue is fixed. Harness now supports adding default values when configuring services, environments, templates, and pipelines.

Reviewer: ```suggestion
  This issue is fixed. Harness now supports adding and editing runtime input default values when configuring services, environments, templates, and pipelines.
```

- The error message for webhook trigger registration failure was unclear. (CDS-53600)

  This issue is fixed by improving the error handling for webhook trigger registration. The error message now conveys a proper error summary.

> - An `IllegalArgumentException` appeared when service variable expressions were used for environment reference. (CDS-53490)

Reviewer: ```suggestion
- An `IllegalArgumentException` appeared when service variable expressions were references in environments. (CDS-53490)
```

>   You should not use service variables for environment reference, and environment variables for service reference. Harness has now improved the error handling mechanism for such scenarios so that users can fix the issue themselves. 

Reviewer: ```suggestion
  You should not reference environment variables in service settings because the environment settings are resolved after the service settings during pipeline execution. Harness has now improved the error handling mechanism for such scenarios. 
```
